src/Sylius/Bundle/CoreBundle/Tests/Controller/ShippingCategoryApiTest.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+namespace Sylius\Bundle\CoreBundle\Tests\Controller;
+
+use Lakion\ApiTestCase\JsonApiTestCase;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class ShippingCategoryApiTest extends JsonApiTestCase
+{
+    public function testCreateShippingCategoryAccessDeniedResponse()
+    {
+        $this->client->request('POST', '/api/shipping-categories/');
+
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'authentication/access_denied', 401);
+    }
+
+    public function testCreateShippingCategoryValidationFailResponse()
+    {
+        $this->loadFixturesFromFile('api_administrator.yml');
+
+        $this->client->request('POST', '/api/shipping-categories/', [], [], [
+            'HTTP_Authorization' => 'Bearer SampleTokenNjZkNjY2MDEwMTAzMDkxMGE0OTlhYzU3NzYyMTE0ZGQ3ODcyMDAwM2EwMDZjNDI5NDlhMDdlMQ',
+            'CONTENT_TYPE' => 'application/json',
+        ]);
+
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'shipping_category/create_validation_fail', 400);
+    }
+
+    public function testCreateShippingCategoryResponse()
+    {
+        $this->loadFixturesFromFile('api_administrator.yml');
+
+        $this->client->request('POST', '/api/shipping-categories/', [], [], [
+            'HTTP_Authorization' => 'Bearer SampleTokenNjZkNjY2MDEwMTAzMDkxMGE0OTlhYzU3NzYyMTE0ZGQ3ODcyMDAwM2EwMDZjNDI5NDlhMDdlMQ',
+            'CONTENT_TYPE' => 'application/json',
+        ], '{"name": "Regular", "description": "Regular weight items", "code": "reg"}');
```

Comment:
https://github.com/lchrusciel/Sylius/pull/1

Also, ``code`` should always be first, then name, description etc.